src/bokeh/plotting/_plot.py

```diff
@@ -162,6 +165,7 @@ def _get_axis_class(axis_type: AxisType | None, range_input: Range, dim: Dim) ->
                 if Datetime.is_timestamp(value):
                     return LinearAxis, {}
                 Datetime.validate(Datetime(), value)
+                # ToDo timedelta
```

Comment:
Just for code grepping consistency
```suggestion
                # TODO: timedelta
```

Reply:
fixed